features/messages/requests/MarkAllReadButton.tsx

```diff
@@ -1,5 +1,4 @@
-import { Tooltip } from "@material-ui/core";
-import IconButton from "components/IconButton";
+import Button from "components/Button/Button";
```

Comment:
```suggestion
import Button from "components/Button";
```
The folder has a default import in `index`